### ISSUE TITLE
Detailed age distribution

### ISSUE THREAD

**Issue by LeonDLotter** [predictive-clinical-neuroscience/braincharts#2]
Dear developers,

I am currently working with your cortical thickness models in the context of multiple projects. Thank you a lot for your amazing work!  
A question that comes naturally when others look into this work concerns the age distribution of the dataset (`lifespan_57K_82sites`). It would be great if I could include a figure of the age distribution (i.e., a histogram) in the manuscripts that will hopefully result from my analyses. I know of the plots in you eLife paper (Figure 1A & B) and given the open access policy of eLife it might actually be possible to include the exact same plots. However, I was wondering whether you could provide me (or the community, respectively) with somewhat more detailled age information?

For me, for example, the distribution near to the minimum age is especially relevant and that is hard to infer from the eLife plots. I assume that you most likely cannot just share a vector with ages for the whole dataset. But would it be possible for you to share a file with the numbers of subject per year (e.g., from 1 to 100 years or so, corresponding to histogram bins)? 

Thank you very much!

Best wishes,
Leon

**Comment by saigerutherford** [comment]
Hi @LeonDLotter. Sorry for the slow reply, but I have added a file called `all_age_eLife.csv` to the `docs/` folder. This includes the ages of all subjects used. There is a column called test_label to show which test set they belong to. If you want to show the age distribution of just the training set, you should filter this file like this: 
`import pandas as pd`
`df = pd.read_csv('all_age_eLife.csv')`
`df_train = df.query('test_label == "controls_train"')`

**Comment by LeonDLotter** [comment]
That's great and very helpful. Thanks a lot!
